.travis.yml

```diff
@@ -5,14 +5,20 @@
 # code is distributed under the terms of the BSD 3-Clause License.
 # The full license text can be found in the LICENSE.txt file.
 #
-
+sudo: false
```

Comment:
Keep the blank line between the comment and code/config.
Opting into travis' container infra, will need to become a commit on its own.